deploy/helm/sumologic/values.yaml

```diff
@@ -2210,48 +2018,21 @@ kube-prometheus-stack:
               sourceLabels: [job, __name__]
         ## prometheus operator rules
         ## :kube_pod_info_node_count:
-        ## :node_cpu_saturation_load1:
-        ## :node_cpu_utilisation:avg1m
-        ## :node_disk_saturation:avg_irate
-        ## :node_disk_utilisation:avg_irate
-        ## :node_memory_swap_io_bytes:sum_rate
-        ## :node_memory_utilisation:
-        ## :node_net_saturation:sum_irate
-        ## :node_net_utilisation:sum_irate
         ## cluster_quantile:apiserver_request_duration_seconds:histogram_quantile
         ## cluster_quantile:scheduler_binding_duration_seconds:histogram_quantile
         ## cluster_quantile:scheduler_framework_extension_point_duration_seconds:histogram_quantile
         ## cluster_quantile:scheduler_e2e_scheduling_duration_seconds:histogram_quantile
         ## cluster_quantile:scheduler_scheduling_algorithm_duration_seconds:histogram_quantile
```

Comment:
Can you also disable the default recording rules under `kube-prometheus-stack.defaultRules`? Then we can get rid of this whole remote write definition.